4_advanced_usage/30_technique_creation.txt

```diff
@@ -68,7 +68,7 @@ package so you should never modify anything here: it will be lost at some point.
 * /var/rudder/configuration-repository/ncf
 ** This is were you add your own ncf Generic Methods and Techniques.
 Techniques created with the Technique Editor will be located here, and both Generic Methods
-and Techniques in that place will be accessible in the Technique Editor alongs what is 
+and Techniques in that place will be accessible in the Technique Editor along what is 
```

Comment:
So... "alongs" is definitely wrong, I agree. But it doesn't really make sense anyhow. This should probably we "alongside".